src/tools/perf/mad/perftest_mad.c

```diff
@@ -551,10 +551,9 @@ static ucs_status_t perftest_mad_accept(perftest_mad_rte_group_t *rte_group,
     lid = rte_group->dst_port.lid;
     ucs_debug("MAD: accept: remote lid:%d/0x%02x", lid, lid);
 
-    /* Import receive params, preserving passed flags */
-    needed_flags = ctx->params.super.flags & UCX_PERF_TEST_FLAG_ERR_HANDLING;
+    backup_local_params(&ctx->params, &params_backup);
```

Comment:
Unfortunately this will break runtime as perftest_mad.c is bundled in libperftest_mad.so, and the loader will fail to resolve `backup_local_params()`/`restore_local_params()`. We are in the process of adding MAD-based runs in CI to catch those issues in the future.

Could you please move the added function in the corresponding header as static inlines?

Reply:
fixed